tests/core/FS_exports_assert_2.out

```diff
@@ -1 +1 @@
-was not exported. add it to EXPORTED_RUNTIME_METHODS (see the FAQ). Alternatively, forcing filesystem support (-s FORCE_FILESYSTEM=1) can export this for you
+was not exported. add it to EXPORTED_RUNTIME_METHODS (see the FAQ). Alternatively, forcing filesystem support (-s FORCE_FILESYSTEM=1) can export this for you)
```

Comment:
This also looks unbalanced, so the full text would be good here as well.

Reply:
So annoying.. I somehow lost those changes.. no idea how.

Restored these now.